pkg/util/trivy/docker.go

```diff
@@ -52,9 +57,85 @@ func convertDockerImage(ctx context.Context, client client.ImageAPIClient, imgMe
 		_ = os.Remove(f.Name())
 	}
 
-	return &image{
+	img := &image{
 		opener:  imageOpener(ctx, DockerCollector, imageID, f, client.ImageSave),
 		inspect: inspect,
 		history: configHistory(history),
-	}, cleanup, nil
+	}
+	img.Image = img
```

Comment:
`img.Image == img` ? that feels strange

Reply:
I don't remember why. Removed it to try as it indeed looks strange